app/src/common/shared/org/mozilla/vrbrowser/telemetry/GleanMetricsService.java

```diff
@@ -36,8 +36,7 @@ public static void init(Context aContext) {
         } else {
             GleanMetricsService.stop();
         }
-        Configuration config = new Configuration(Configuration.DEFAULT_TELEMETRY_ENDPOINT,
-                                                 BuildConfig.BUILD_TYPE);
+        Configuration config = new Configuration();
```

Comment:
This was just fixed in mozilla-mobile/android-components#4892. @pocmo is cutting a new Android Components 19 dot release today, so you'll be able to jump to the new version and restore the build type.